### PR TITLE
[Bots][Universalbotactionmodel][1149654]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Created draft version of Universal bot action model document. Please review.

### CODE REVIEW

File: msteams-platform/bots/Universal-bot-action-model/Work-with-universal-bot-action-model.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    > Whenever your bot returns a new card as a result of processing an `Action.Execute`, the response must conform to the [response format](#response-format).
+
+## Schema for universal bot action model
+
+The universal bot action model is introduced in the adaptive cards schema version 1.4. To use the adaptive card effectively, the `version` property of your adaptive card must be set to 1.4 or higher.
+
+> [!NOTE]
+> Setting the `version` property to 1.4 makes your adaptive card incompatible with older clients of the platforms or applications, such as Outlook and Teams, as they do not support the universal bot action model.
+
+If you set the card version to less than 1.4 and use either or both, `refresh` property and the `Action.Execute` command the following happens:
```
Comment (sowrabh-msft) [inline msteams-platform/bots/Universal-bot-action-model/Work-with-universal-bot-action-model.md:38]: Let us use `action` instead of `command` everywhere

**Review by sowrabh-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/bots/Universal-bot-action-model/Overview.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+### Data-driven views
+
+Data-driven views refer to a model where cards are defined using the following two components:
+
+* An adaptive card template, designed to bind data conforming to a specific format.
+* The data to bind the template.
+
+### Sequential workflow support
+
+With sequential workflow, any user part of a chat or channel can take action on their specific adaptive card. This workflow allows the user to take action or respond to cards without modifying the card for other users.
```
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/bots/Universal-bot-action-model/Overview.md:86]: Could you please elaborate on this comment. Should I rephrase the statements.

File: msteams-platform/bots/Universal-bot-action-model/Overview.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: Overview of universal bot action model
+description: A quick overview of universal bot action model.
+ms.topic: overview
+---
+
+# Universal bot action model
```
Comment (ojasvichoudhary) [inline msteams-platform/bots/Universal-bot-action-model/Overview.md:7]: Universal Actions for adaptive cards

**Review by ojasvichoudhary — DISMISSED** [review]

**Review by shiladityasaha — DISMISSED** [review]
I have made some minor changes across pages. Please check and merge those. Approving the PR.

File: msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Today every user in the Teams chat or channel sees the exact same view and button actions on the Adaptive Card. However, in certain scenarios there is a requirement for certain users to act differently and have access to different information within the same chat or channel.
+
+For example, if you send an incident reporting card in a chat or channel, only the user who is assigned the incident must see a **Resolve** button. On the other hand, the incident creator must see an **Edit** button and all other users must only be able to view details of the incident. This is made possible by user specific views that is enabled by the `refresh` property. 
+
+The following image shows an example of a ticketing messaging extension (ME) where different users in the chat are shown different actions based on the requirement:
+
+:::image type="content" source="~/assets/images/adaptive-cards/universal-bots-incident-management.png" alt-text="User-specific views":::
+
+For more information, see [sample for User Specific Views](User-Specific-Views.md).
```
Comment (sowrabh-msft) [inline msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md:55]: This link seems to be broken. Please check

Comment (sowrabh-msft) [reply msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md:55]: In fact all user-specific-view.md links are broken

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by sowrabh-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved the PR

**Pull request merged**